.ci/apm-beats-update.groovy

```diff
@@ -127,6 +127,7 @@ def beatsUpdate() {
         git config --global --add remote.origin.fetch "+refs/pull/*/head:refs/remotes/origin/pr/*"
 
         go mod edit -replace github.com/elastic/beats/v7=\${GOPATH}/src/github.com/elastic/beats-local
+        echo '{"name": "${GOPATH}/src/github.com/elastic/beats-local", "licenceType": "Elastic"}' >> \${GOPATH}/src/github.com/elastic/beats-local/dev-tools/notice/overrides.json
```

Comment:
playing devils advocat: We have 2 licenses. The OSS apm-server should not be build with Beats Elastic License :)

Reply:
This is not part of the build process, and has zero impact on packaging. This is just a workaround for a job that checks if we added anything to Beats which might possibly break something in APM. Also, it is somewhat a shame that the job has been failing for more than a month and we haven't noticed it. :(

We can change it to Apache, but the text in our LICENCE.txt is not Apache. It states that it has two licences; Elastic and Apache. To me, putting Elastic in this workaround felt more truthful. I can change it regardless, but it seems unnecessary.